tests/test_hypothesis.py

```diff
@@ -95,6 +102,10 @@ def test_equality(d):
 @given(d=d, arg1=immutable, arg2=immutable, itemlist=lists(tuples(immutable, immutable), **sz))
 def test_consistency(arity, methodname, B, d, arg1, arg2, itemlist):
     ordered = issubclass(B, OrderedBidirectionalMapping)
+    if ordered and WORKAROUND_OD_NAN_BUG:
+        #assume(nan not in d and nan not in d.values())
+        # TODO: For some reason the assume does not work. Resort to:
+        d = {(k, v) for (k, v) in iteritems(d) if not isnan_(k) and not isnan_(v)}
```

Comment:
Thanks @DRMacIver, that did the trick!

Speaking of screwy nan behavior, I think I found another bug when nan is used in an OrderedDict:
https://github.com/jab/bidict/blob/0.12.0-dev/tests/test_hypothesis.py#L95-L97
It appears to have been fixed in Python 3.5, but I couldn't find an issue for it in the tracker. Have you seen that one?